ios/Classes/CameraPreview.swift

```diff
@@ -286,6 +294,13 @@ class CameraPreview: NSObject, FlutterPlatformView {
         guard captureSession != nil, let camera = camera, camera.isConnected else {
             throw MlKitPluginError.cameraIsNotInitialized
         }
+        try internalSetZoom(value)
+    }
+
+    private func internalSetZoom(_ value: Double) throws {
```

Comment:
Какой смысл вкладывается в internal?
```suggestion
    private func setZoom(_ value: Double) throws {
```